### PR TITLE
Add documents to `/propertyPack/documents`

### PR DESCRIPTION
**Pull request by edmolyneux**
Here's an implementation of the documents properties additions for review please

### CODE REVIEW

**Comment by amolpednekar** [comment]
@edmolyneux - How is this is different to `documented/eletronic vouches` ? That's the place I envisioned any attached documents would reside. 

Is it specifically to map the "document" fields we find on various forms?

File: src/schemas/v2/combined.json [hunk 1]
```diff
@@ -36632,6 +36632,39 @@
...
+                "title": "Document MIME type",
+                "type": "string",
+                "minLength": 1
+              },
+              "documentType": {
+                "title": "Human-readable document category",
+                "type": "string",
+                "minLength": 1
+              },
+              "externalIds": {
```
Comment (vivek656) [inline src/schemas/v2/combined.json:36663]: Where is the actual document stored, 
should we either provide a field for inline base64 document (like in verified claims)
or provide URL for the document, 

OPTIONAL Addition Need to discuss-> with URL can add fields to provide basic auth type (that signifies, if URL is publicly available or need some Auth).

Comment (amolpednekar) [reply src/schemas/v2/combined.json:36663]: And still keep an additional "externalReference" `string` or `object` to capture custom metadata

**Review by amolpednekar — CHANGES_REQUESTED** [review]
Added comments

File: src/schemas/v2/combined.json (hunk 1 above)
Comment (edmolyneux) [reply src/schemas/v2/combined.json:36663]: Our view is that the verification + evidence part of the claim provides the location of the file

**Review by amolpednekar — APPROVED** [review]
LGTM

File: src/schemas/v2/pdtf-transaction.json [hunk 2]
```diff
@@ -33248,6 +33248,39 @@
...
+                "title": "Document MIME type",
+                "type": "string",
+                "minLength": 1
+              },
+              "documentType": {
+                "title": "Human-readable document category",
+                "type": "string",
+                "minLength": 1
+              },
+              "externalIds": {
```
Comment (vivek656) [inline src/schemas/v2/pdtf-transaction.json:33279]: Why external ids are left as a object  this makes the schema values non deterministic, can we make external ids as list of strings. In place of just objects  .

**Pull request merged**